src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -132,25 +132,36 @@ class Microsoft_Launcher : public PowertoyModuleIface {
    // Enable the powertoy
   virtual void enable()
   {
+      unsigned long powertoys_pid = GetCurrentProcessId();
+
       if (is_process_elevated(false) == false)
       {
+          std::wstring executable_args = L"";
+          executable_args.append(std::to_wstring(powertoys_pid));
+
           SHELLEXECUTEINFOW sei{ sizeof(sei) };
           sei.fMask = { SEE_MASK_NOCLOSEPROCESS | SEE_MASK_FLAG_NO_UI };
           sei.lpFile = L"modules\\launcher\\PowerLauncher.exe";
           sei.nShow = SW_SHOWNORMAL;
+          sei.lpParameters = executable_args.data(); 
           ShellExecuteExW(&sei);
 
           m_hProcess = sei.hProcess;
       }
       else
       {
+          std::wstring executable_args = L"";
+          executable_args.append(L"-start_PowerLauncher");
+          executable_args.append(L" ");
+          executable_args.append(std::to_wstring(powertoys_pid));
+
           std::wstring action_runner_path = get_module_folderpath();
           action_runner_path += L"\\action_runner.exe";
           SHELLEXECUTEINFOW sei{ sizeof(sei) };
           sei.fMask = { SEE_MASK_NOCLOSEPROCESS | SEE_MASK_FLAG_NO_UI | SEE_MASK_NOASYNC };
           sei.lpFile = action_runner_path.c_str();
           sei.nShow = SW_SHOWNORMAL;
-          sei.lpParameters = L"-start_PowerLauncher";
+          sei.lpParameters = executable_args.data();
```

Comment:
This code will be effected by the hotfix for 0.18.2, so let's wait for the hotfix before merging these changes.